torch/ao/quantization/fx/convert.py

```diff
@@ -59,6 +60,27 @@
 )
 from .lower_to_fbgemm import lower_to_fbgemm
 
+
+# TODO: revisit this list. Many helper methods shouldn't be public
+__all__ = [
+    "convert",
+    "convert_custom_module",
+    "convert_standalone_module",
+    "convert_weighted_module",
+    "duplicate_dequantize_node",
+    "duplicate_quantize_dynamic_node",
+    "get_module_path_and_prefix",
+    "has_none_qconfig",
+    "insert_dequantize_node",
+    "maybe_get_observer_for_node",
+    "maybe_recursive_remove_dequantize",
+    "remove_extra_dequantize",
+    "remove_quant_dequant_pairs",
+    "restore_state",
+    "run_weight_observers",
```

Comment:
In this PR I just wanted to preserve the existing behavior for unrelated functions. We should further restrict the visibility here and other places but let's leave that for a future PR.